api/envoy/extensions/filters/network/ext_authz/v3/ext_authz.proto

```diff
@@ -62,5 +62,5 @@ message ExtAuthz {
   //
   // When this field is true, Envoy will send an HTTP response code appropriate to
   // the response seen
```

Comment:
I think  we should update this comment to say that when this is set to true, the downstream HTTP response code will match the HTTP response code observed from the authorization server instead of 200 OK. We should also clarify whether grpc-status (and other grpc headers) are still maintained, since I think the outcome of using this flag is a sort of mixed-mode gRPC/HTTP reply (some semantics of grpc, some of HTTP 1?)

Reply:
I did update this somewhat in my last push, based on @rojkov's comment but can clarify it further w.r.t. gRPC vs HTTP.

Reply:
I've updated the comments, as suggested.  What is the process for updating the docs?  I didn't see a reference in the contributing reference, but may have just missed it.

Thanks for the extensive and speedy reviews!

Reply:
The API docs get updated automatically upon changes in proto files. For example the doc page generated from this proto is https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto#envoy-v3-api-msg-extensions-filters-http-ext-authz-v3-extauthz.

To check how a doc page would look like after your changes you can generate it locally with `./docs/build.sh`.

Reply:
Thanks @rojkov - I've updated the proto notes based on the generation results.

Reply:
`links -dump  generated/docs/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto.html |grep -A9 retain_http_status_on_grpc_deny` 

-->

```
   retain_http_status_on_grpc_deny

           (bool) Specifies whether HTTP status codes for gRPC calls are
           passed intact from the called service back to the caller on gRPC
           DENIED.

           When this field is true, Envoy’s HTTP response to gRPC calls which
           receive a DENIED response will retain the status code from the
           called service’s HTTP headers. The gRPC headers are not affected.
```